include/vecmath/mat.h

```diff
@@ -826,7 +826,7 @@ namespace vm {
      * matrix is invertible, and if so, the matrix is the inverted given matrix
      */
     template <typename T, std::size_t S>
-    constexpr std::tuple<bool, mat<T, S, S>> invert(const mat<T, S, S>& m) {
+    constexpr std::tuple<bool, mat<T, S, S>> invert_cramer(const mat<T, S, S>& m) {
```

Comment:
If your code performs better and has no other drawbacks, we can remove this for all I care.